src/main/java/LacksDataBaseProject/Item.java

```diff
@@ -0,0 +1,50 @@
+package LacksDataBaseProject;
+
+public class Item {
+    protected String itemID;
+    protected String itemName;
+
+    Item(String itemID, String itemName) {
+        if (itemID == null || itemID.isEmpty()) {
+            throw new NullPointerException("itemID can't be empty");
+        } else {
+            this.itemID = itemID;
+        }
+        if (itemName == null || itemName.isEmpty()) {
```

Comment:
tutaj bardziej by się przydało coś takiego jak MissingArguemtException :) bo to nie jest jeszcze null, a raczej brak pól który umożlwią prawidłowe stworzenie obiektu :)

Reply:
ok